aiokafka/record/legacy_records.py

```diff
@@ -73,6 +76,20 @@ class LegacyRecordBase:
     LOG_APPEND_TIME = 1
     CREATE_TIME = 0
 
+    def _assert_has_codec(self, compression_type):
```

Comment:
Can we reuse the same function to avoid copy-paste?

Reply:
Sure. What module is the best place to put in?

Reply:
Oh, sorry. They will become different after adding zstd support. I think that's the reason why it's duplicated in kafka-python.

Reply:
According to [KIP-110](https://cwiki.apache.org/confluence/display/KAFKA/KIP-110:+Add+Codec+for+ZStandard+Compression#KIP110:AddCodecforZStandardCompression-RejectedAlternatives)
> Rejected Alternatives
> A. Support ZStandard to the old clients which can understand v0, v1 messages only.